packages/icons/lib/icon-template.cjs

```diff
@@ -0,0 +1,13 @@
+// index-template.js
+let path = require('path')
+
+function defaultIndexTemplate(filePaths) {
+  const exportEntries = filePaths.map((filePath) => {
+    const basename = path.basename(filePath, path.extname(filePath))
+    const exportName = /^\d/.test(basename) ? `Svg${basename}` : basename
+    return `export { default as ${exportName}Icon } from './${basename}'`
```

Comment:
Mabe better to export icons as `Icon${basename}` rather than the other way around? Then the occasional `Svg` prefix 1 line above can be avoided.